# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../crm114/mailreaver_text_cache_location.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./mailreaver.crm.orig	2008-03-26 08:50:45.000000000 -0400
+++ ./mailreaver.crm	2008-08-13 10:42:03.000000000 -0400
@@ -174,6 +174,12 @@
     output [:*:fileprefix::*:log_all_mail_to_file:] <append> /:*:_dw:/
 }
 
+#     :text_cache: has to be :fileprefix: relative
+{
+    match [:text_cache:] /./
+    alter (:text_cache:) /:*:fileprefix::*:text_cache:/
+}
+
 
 ########################################################################
 #
